challenges/objects-arrays.js

```diff
@@ -80,8 +121,11 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 let lowerCase = [];
+
```

Comment:
Because .map creates an array, you can immediately use it as the variable and remove the brackets. 

let lowerCase = zooAnimals.map(function(animalName) {
  return animalName.animal_name.toLowerCase();
})
console.log(lowerCase);

Here you pass a parameter into map method, animalName (though you could call it anything).  Use the "dot" binding to connect this parameter to the animal_name key.  toLowerCase will then access every value on this key and make it lower case.